USERGUIDE.md

```diff
@@ -75,6 +75,7 @@ Config.userDefaults?.set("MY_CUSTOM_ID", forKey: Config.Key.subscriptionKey.rawV
 * [Load the Mini App list](#load-miniapp-list)
 * [Get a MiniAppInfo](#get-mini-appinfo)
 * [Getting a MiniApp meta-data](#get-mini-meta-data)
+* [How to use MiniApp meta-data](#how-to-use-meta-data)
```

Comment:
there should be a MiniApp meta-data section with these 2 subsections

Reply:
Done